src/config-import.js

```diff
@@ -65,10 +57,12 @@ export async function readConfig(configPath) {
       /** @type {IconData | undefined} */
       let icon
 
-      iconsZip = await yauzl.open(configPath)
-      const entries = await iconsZip.readEntries(MAX_ENTRIES)
-      for (const entry of entries) {
-        if (!entry.filename.match(/^icons\/([^/]+)$/)) continue
+      // we sort the icons by filename so we can group variants together
+      const iconEntries = entries
+        .filter((entry) => entry.filename.match(/^icons\/([^/]+)$/))
```

Comment:
```suggestion
        .filter((entry) => entry.filename.startsWith('icons/')
```

Fine as it is, just not sure the regex is necessary

Reply:
The difference is that with the regex you only match files in the icon folder, where with the other option you will also match with 'icons/' (so the listing of the directory itself) which we don't want. Instead, we could do:
```js
.filter(
          (entry) =>
            entry.filename.startsWith('icons/') && !entry.filename.endsWith('/')
        )

```
but I dunno...

Reply:
Yeah, just keep the regex